src/h.js

```diff
@@ -337,13 +337,73 @@ const hashMatch = (str) => { return window.location.hash === str; },
         };
     },
 
-    ga = (eventObj, fn = Function.prototype) => {
-        const ga = window.ga || {};
+    analyticsEvent = (eventObj, fn) => {
+        //https://developers.google.com/analytics/devguides/collection/analyticsjs/command-queue-reference#send
+        if(!eventObj)
+          return fn;
 
         return () => {
-            ga('send', eventObj);
-            fn();
+          console.log('event',eventObj);
+          try {
+            var dataProject = eventObj && eventObj.project ? {
+              project_id:eventObj.project.id,
+              category_id: eventObj.project.category_id,
+              state: eventObj.project.address && eventObj.project.address.state_acronym,
+              city: eventObj.project.address && eventObj.project.address.city
+            } : null;
+            var dataUser = eventObj && eventObj.user ? {} : null;//TODO
+            var data = _.extend({},dataProject, dataUser,eventObj && eventObj.extraData);
+
+            const ga = window.ga;//o ga tem q ser verificado aqui pq pode não existir na criaçaõ do DOM
+            if(eventObj && ga && ga.getAll) {
+              //https://developers.google.com/analytics/devguides/collection/analyticsjs/sending-hits#the_send_method
+              ga('send', 'event', eventObj.cat, eventObj.act, eventObj.lbl, {
+                nonInteraction: eventObj.nonInteraction!==false,//default é true,e só será false se, e somente se, esse parametro for definido como false
+                transport: 'beacon',
+                hitCallback: fn && (() => { fn(); })//coloquei dentro de outra função para fn não receber params
+              });
+            } else {
+              fn && fn();
```

Comment:
We could replace those short-circuits evaluations by using es2015 default parameter. Ex.: